### PR TITLE
Add uuid test

### PR DESCRIPTION
**Pull request by iyear**
#### What type of PR is this?

test

#### What this PR does / why we need it:

Increase coverage

#### Which issue(s) this PR fixes:

#2167

### CODE REVIEW

**Review by kannon92 — COMMENTED** [review]
Thank you for the PR. You will have to run our linter step but otherwise it looks good to me

**Comment by iyear** [comment]
Updated. Thanks for your review.

File: internal/executor/util/uuid_test.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+)
+
+func TestStringUuidsToUuids(t *testing.T) {
+	tests := []struct {
+		name        string
+		uuidStrings []string
+		want        []armadaevents.Uuid
+		wantErr     bool
+	}{
+		{"test1", []string{"1", "2", "3"}, []armadaevents.Uuid{}, true},
```
Comment (d80tb7) [inline internal/executor/util/uuid_test.go:16]:
```suggestion
		{"invalid uuid", []string{"1", "2", "3"}, []armadaevents.Uuid{}, true},
```

File: internal/executor/util/uuid_test.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+		{"test2", []string{"52a3cfa6-8ce1-42b1-97cf-74f1b63f21b9"}, []armadaevents.Uuid{{5954831446549021361, 10939090601399755193}}, false},
```
Comment (d80tb7) [inline internal/executor/util/uuid_test.go:17]:
```suggestion
		{"valid uuid", []string{"52a3cfa6-8ce1-42b1-97cf-74f1b63f21b9"}, []armadaevents.Uuid{{5954831446549021361, 10939090601399755193}}, false},
```

Comment (iyear) [reply internal/executor/util/uuid_test.go]: Done.

File: internal/executor/util/uuid_test.go (hunk 1 above)
Comment (iyear) [reply internal/executor/util/uuid_test.go]: Done.

**Review by kannon92 — APPROVED** [review]

**Pull request merged**